microsoft-edge/webview2/how-to/set-preview-channel.md

```diff
@@ -6,259 +6,410 @@ ms.author: msedgedevrel
+
+
+##### [Registry key](#tab/registry-key)
 
 1. Open a PowerShell terminal or PowerShell-enabled command prompt.
 
 1. Modify and then run the following command:
 
-   `REG ADD HKLM\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannelPreference /v WebView2APISample.exe /t REG_SZ /d "1"`
+   ```reg
+   REG ADD <HKLM/HKCU>\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannels /v WebView2APISample.exe /t REG_SZ /d "0,1"
+   ```
+
+Replace `WebView2APISample.exe` with your own app executable name or the application user model ID.  Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine, and can result in unexpected behavior.
+
+
+##### [Environment variable](#tab/environment-variable)
 
-   Replace `WebView2APISample.exe` with the file name of your own app executable or the Application User Model ID. Using a wildcard (*) as the value name will apply the override to _all_ WebView2 apps on the machine and can result in unexpected behavior.
+Name: `WEBVIEW2_RELEASE_CHANNELS`  <!-- 2 trailing spaces = linebreak -->
+Value: `<string of desired channels (ex: 0,1)>`
 
-#### Resuming using the default, WebView2 Evergreen Runtime
+The target app can't be set when using the environment variable approach.  Therefore, when set as a global environment, this setting will affect all apps that use WebView2 on the machine.
 
-To delete the `ReleaseChannelPreference` registry override, run the command:
 
-`REG DELETE HKLM\Software\Policies\Microsoft\Edge\WebView2\ReleaseChannelPreference /f`
+##### [Group policy](#tab/group-policy)
 
+Set the `ReleaseChannel` policy.
+
+* Name: `<app exe name or app user model ID - ex: WebView2APISample.exe>`  <!-- 2 trailing spaces = linebreak -->
+* Value: `<string of desired channels (ex: 0,1)>`
+
+Do either of the following:
+
+* Download the Microsoft Edge policy files, which include the WebView2 policy files, from [Download and configure Microsoft Edge for Business](https://www.microsoft.com/edge/business/download).  For more information, see [Configure Microsoft Edge policy settings on Windows devices](/deployedge/configure-microsoft-edge).
+
+* Use the built-in policy on Intune.  For more information, see [Configure Microsoft Edge policy settings in Microsoft Intune](/mem/intune/configuration/administrative-templates-configure-edge).
+
+---
 
-<!-- 3. Environment Variable ============================================== -->
-## Using an environment variable
+<!-- ------------------------------ -->
+#### How to use an environment variable
 
-To make your application use a Microsoft Edge preview channel by using an environment variable:
 
-1. In the Windows search bar, enter "environment", and then select **Edit the system environment variables**.
+<!-- ---------- -->
+###### Setting environmental variables using API
+
+You can set environment variables by using `SetEnvironmentVariable`.
```

Comment:
```suggestion
You can set environment variables by using `SetEnvironmentVariable`. If you call this function from your host app, it will limit the scope of the environment variable to just your host app and the WebView2 controls it creates, and won't affect other apps on the system.
```

Reply:
```
You can set environment variables by using `SetEnvironmentVariable`.  
Calling this function from your host app limits the scope of the environment variable 
to just your host app and the WebView2 controls that it creates, 
and doesn't affect other apps on the system.
```